### DISCUSSION TITLE
Deprecation notice on binary sensor.

### DISCUSSION THREAD

**Discussion by droyholmes** [tetele/onju-voice-satellite#99] [Q&A]
I believe I'm using the original code from the PCB designer on my onju. I recently got the message "The ESPHome in progress binary sensor binary_sensor.onju_voice_satellite_01_24b0f0_assist_in_progress is deprecated." Does this project also get that warning? If not, are there any "gotcha"s I should be aware of when trying to switch over?

**Comment by tetele** [comment]
Regardless of what config you use, that warning has to do with a change in HA from a couple of months ago and you can safely ignore it.
